implementations/rs/tests/it/main.rs

```diff
@@ -0,0 +1,30 @@
+use polywrap_http_plugin::HttpPlugin;
+use polywrap_client::client::PolywrapClient;
+use polywrap_core::{
+    client::ClientConfig,
+    uri::Uri,
+};
+use polywrap_resolvers::static_resolver::{StaticResolver, StaticResolverLike};
+use polywrap_plugin::{package::PluginPackage};
+use std::{sync::Arc};
+
+
+mod get;
+mod post;
+
+pub fn get_client() -> PolywrapClient {
```

Comment:
I like that we can now use the Rust client and don't have to rely on the JS client for rust wrappers.
Though, I think for most cases we could just mock the subinvocations(if any) and have unit tests instead of e2e